llvm/utils/TableGen/RegisterBankEmitter.cpp

```diff
@@ -287,8 +291,88 @@ void RegisterBankEmitter::emitBaseClassImplementation(
      << "  for (auto RB : enumerate(RegBanks))\n"
      << "    assert(RB.index() == RB.value()->getID() && \"Index != ID\");\n"
      << "#endif // NDEBUG\n"
-     << "}\n"
-     << "} // end namespace llvm\n";
+     << "}\n";
+
+  uint32_t NoRegBanks = Banks.size();
```

Comment:
very very tiny nit, but at first I thought this meant "no reg bank" as if it's a negative.
Maybe use `N` or `Num`?